winui/Avatar-view/visual-styles.md

```diff
@@ -48,17 +48,17 @@ avatarView.CornerRadius = new CornerRadius(0,23,0,23);
 
 ## Circle
 
-For the circle style, use the `AvatarSize` property to set the avatar's pre-defined circle size, which includes the following values.
+For the circle style, use the [AvatarSize](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Core.SfAvatarView.html#Syncfusion_UI_Xaml_Core_SfAvatarView_AvatarSize) property to set the avatar's pre-defined circle size, which includes the following values.
 
 * Extra-large
 * Large
 * Medium
 * Small
 * Extra-small
 
-The following code sample illustrates how to set the visual styles of AvatarView to `Circle`.
+The following code sample illustrates how to set the visual styles of AvatarView to [Circle](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Core.AvatarShape.html#Syncfusion_UI_Xaml_Core_AvatarShape_Circle).
 
-N> The default `AvatarShape` is Circle.
+N> The default AvatarShape is Circle.
```

Comment:
@MadhumithanJayapathi Add link for notes section.

Reply:
@Manivannan-E Added links for notes section.